### PR TITLE
feat(service): Add Amazon SES service

### PR DESCRIPTION
**Pull request by chanioxaris**
Added support for Amazon Simple Email Service.

### CODE REVIEW

**Comment by checkaayush** [comment]
@chanioxaris You might want to consider using AWS Go SDK v2 (https://github.com/aws/aws-sdk-go-v2). This has a lot of improvements on the older SDK.

**Comment by chanioxaris** [comment]
> @chanioxaris You might want to consider using AWS Go SDK v2 (https://github.com/aws/aws-sdk-go-v2). This has a lot of improvements on the older SDK.

Thanks, didn't know that there was a newer version of Golang SDK. Updated my implementation.

**Comment by nikoksr** [comment]
@chanioxaris appreciate your contribution! Great stuff. And thanks @checkaayush for the info about the up-to-date SDK.

@chanioxaris basically ready to merge. I just need to know if you have successfully tested the service under real conditions.

**Comment by chanioxaris** [comment]
> @chanioxaris basically ready to merge. I just need to know if you have successfully tested the service under real conditions.

Hello @nikoksr, yes i have tested using my own credentials and the message delivered to every receiver email address

File: service/amazonses/amazon_ses.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+				Data: aws.String(message),
+			},
+			// Text: &types.Content{
+			//     Data:    aws.String(message),
+			// },
+		},
+		Subject: &types.Content{
+			Data: aws.String(subject),
+		},
+	}
```
Comment (nikoksr) [inline service/amazonses/amazon_ses.go:69]:
```suggestion
	input := &ses.SendEmailInput{
		Source: a.senderAddress,
		Destination: &types.Destination{
			ToAddresses: a.receiverAddresses,
		},
		Message: &types.Message{
			Body: &types.Body{
				Html: &types.Content{
					Data: aws.String(message),
				},
				// Text: &types.Content{
				//     Data:    aws.String(message),
				// },
			},
			Subject: &types.Content{
				Data: aws.String(subject),
			},
		},
	}
```

Just a change of style. I think it looks a little cleaner this way.

**Review by nikoksr — CHANGES_REQUESTED** [review]
Very well done. Just a tiny change to make. Really appreciate your contribution!

File: service/amazonses/amazon_ses.go (hunk 1 above)
Comment (chanioxaris) [reply service/amazonses/amazon_ses.go]: Sure, i did the minor refactor

**Review by nikoksr — APPROVED** [review]

**Pull request merged**